alkira/resource_alkira_connector_aws_vpc_helper.go

```diff
@@ -77,3 +77,27 @@ func generateUserInputPrefixes(cidr []interface{}, subnets *schema.Set) ([]alkir
 
 	return prefixes, nil
 }
+
+// expandAwsVpcTgwAttachments expand tgw_attachment of connector_aws_vpc
+func expandAwsVpcTgwAttachments(in *schema.Set) []alkira.TgwAttachment {
```

Comment:
I think these expand helper functions merit unit tests. It's technically a simple conversion, but mistakes happen and the test is easy to write.